app/models/concerns/labware.rb

```diff
@@ -4,16 +4,19 @@
 module Labware
   extend ActiveSupport::Concern
 
+  # turn into constant as enum no longer works
+  # allows for a different prefix for each labware type
+  ID_PREFIXES = { plate: 1, tube: 2 }.freeze
+
   included do
-    enum id_prefix: { plate: 1, tube: 2 }
```

Comment:
It looks like it now requires enum to be have a db column so you get the error can't find id_prefix. It was just as easy to change it to a hash rather than try and continue using enum.